samples/openapi3/client/petstore/python-experimental/petstore_api/model_utils.py

```diff
@@ -204,6 +208,10 @@ def to_str(self):
         """Returns the string representation of the model"""
         return pprint.pformat(self.to_dict())
 
+    def __hash__(self):
+        """Returns the hash value of this object."""
+        return hash(frozenset(self._data_store.items()))
+
     def __eq__(self, other):
         """Returns true if both objects are equal"""
         if not isinstance(other, self.__class__):
```

Comment:
How about using our hash method in out `__eq__` method?

Reply:
> How about using our hash method in out `__eq__` method?

What are you trying to achieve by doing this? hash can have collisions, so it cannot be used as is, we must do an actual equality comparison.

Reply:
My intent was to make our definition consistent with what I was reading in python docs about implementing `__hash__` and `__eq__`. If we have a collision chance, I agree that keeping this as-is is better.